src/deadline/blender_adaptor/BlenderAdaptor/schemas/init_data.schema.json

```diff
@@ -12,6 +12,15 @@
                 "cycles"
             ]
         },
+        "render_device": {
+            "enum": [
+                "NONE",
```

Comment:
Since `NONE` in this instance is always going to mean `CPU` what do you think about this being `CPU` instead?

Reply:
Ahhhhh, I see. Ya I think renaming to something like `gpu_render_device` would go a long way to help with the confusion.